From d5b31d55fa1536a5bd08cf85929ac63a8f723467 Mon Sep 17 00:00:00 2001
From: Jiří Klimeš <jklimes@redhat.com>
Date: Wed, 23 May 2012 08:31:12 +0000
Subject: dbus: remove 'max_replies_per_connection' limit from D-Bus configuration

It is leftover from times when D-Bus default limit was 32. Now, it is 8192, see
http://cgit.freedesktop.org/dbus/dbus/commit/?id=8d3d8ff55739eebd84d0d53a20a025329feafc3b
See also https://bugs.freedesktop.org/show_bug.cgi?id=50264
---
diff --git a/src/org.freedesktop.NetworkManager.conf b/src/org.freedesktop.NetworkManager.conf
index 501b68b..7df1c8e 100644
--- a/src/org.freedesktop.NetworkManager.conf
+++ b/src/org.freedesktop.NetworkManager.conf
@@ -112,7 +112,5 @@
                        send_interface="org.freedesktop.NetworkManager"
                        send_member="wake"/>
         </policy>
-
-        <limit name="max_replies_per_connection">512</limit>
 </busconfig>
 
--
cgit v0.9.0.2-2-gbebe
